CHANGES.md

```diff
@@ -60,6 +60,9 @@ Unreleased
 - Allow `(stdlib ...)` to be used with `(wrapped false)` in library stanzas
   (#7139, @anmonteiro).
 
+- Fix segfault on macos when `Fsevents.stop` is called inside the callback to
+  `Fsevents.create`. (#7312, fixes #6151, @gridbugs, @emillon)
```

Comment:
I'd remove such details from the CHANGE list. It's supposed to make sense to end users that know nothing about dune's code. Perhaps something like:

```
- Fix segfault on MacOS when dune was being shutdown while in watch mode.
```